src/tokens-studio/foundations/spectrum/layout/mobile.json

```diff
@@ -620,17 +620,17 @@
     }
   },
   "corner-radius-75": {
-    "value": "2px",
+    "value": "3px",
     "type": "borderRadius",
     "$extensions": {
       "spectrum-tokens": {
-        "uuid": "ecb9d03a-7340-4b43-8aa7-f89eef9f3a20",
+        "uuid": "cddbb2f6-b3a8-4416-90ed-0d7cd1bc7248",
```

Comment:
this UUID change is wrong. for desktop the value is 3px,too and therefor the UUIDs should matched. It should be `ecb9d03a-7340-4b43-8aa7-f89eef9f3a20` here.